pages/login.tsx

```diff
@@ -12,12 +12,14 @@ import { KernLogo } from "@/pkg/ui/Icons"
 import { DemoFlow } from "@/pkg/ui/DemoFlow"
 import { getValueIdentifier, getValuePassword } from "@/util/helper-functions"
 import ory from "@/pkg/sdk"
-import { isDemoUser, isManagedApp } from "@/util/constants"
+import { MiscInfo } from "@/services/basic-fetch/misc"
 
 const Login: NextPage = () => {
   const [initialFlow, setInitialFlow] = useState<LoginFlow>()
   const [changedFlow, setChangedFlow] = useState<LoginFlow>()
   const [selectedRole, setSelectedRole] = useState<string | undefined>('engineer');
+  const isManaged = MiscInfo.isManaged;
+  const isDemo = MiscInfo.isDemo;
```

Comment:
for the isDemo I used the const because it is used on more places in the component, but for isManaged that's not needed

Reply:
no i mean why in general? the const is just a reference to the slightly longer expression that shouldn't change on reader (since pretty static). Since this is run on every render (which may or may not happen quite often :D ) the reference is reassigned every time. Imo no need to store in const variable :)